packages/design-picker/src/hooks/use-theme-designs-query.ts

```diff
@@ -0,0 +1,77 @@
+import { useQuery, UseQueryResult, UseQueryOptions } from 'react-query';
+import wpcom from 'calypso/lib/wp'; // eslint-disable-line no-restricted-imports
+import { Design } from '../types';
+
+// Ideally this data should come from the themes API, maybe by a tag that's applied to
+// themes? e.g. `link-in-bio` or `no-fold`
+const STATIC_PREVIEWS = [
+	'bantry',
+	'sigler',
+	'miller',
+	'pollard',
+	'paxton',
+	'jones',
+	'baker',
+	'kingsley',
+];
+
+export interface UseThemeDesignsQueryOptions {
+	filter?: string;
+	tier?:
+		| '' // Returns all themes regardless of tier
```

Comment:
I think `all` might be more explicit and better

Reply:
Was trying to preserve the existing behaviour, but `'all'` is more self-documenting.